packages/prometheus/data_stream/collector/fields/fields.yml

```
@@ -1,3 +1,7 @@
+- name: metrics_count
+  type: long
+  description: |
+    Total count of Prometheus metrics within the Elasticsearch document. This value is calculated prior to any enrichments by Ingest Pipelines or Agent Processors, ensuring consistency. Subsequent modifications are not reflected in this count. This field name is reserved for internal use and must not be altered using Agent Processors or Ingest Pipelines.
```

Can we shorten the description

```suggestion
    Total count of Prometheus metrics within the Elasticsearch document. This value is calculated prior to any enrichments by Ingest Pipelines or Agent Processors, ensuring consistency. This field name is reserved for internal use and must not be altered using Agent Processors or Ingest Pipelines.
```